### PR TITLE
FW-4520 Use Stats API

### PR DESCRIPTION
**Pull request by bronson-c**
### Description of Changes
This PR re-enables the stats widget and connect it to the new stats API in the Django BE.

### Checklist

- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [ ] Tests have been updated / created if applicable

### Reviewers Should Do The Following:

- [x] Review the changes here
- [ ] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: src/common/dataHooks/useStats.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      totals[type] = getPeriodTotalForType(period, type)
+    })
+    return totals
+  }
+
+  const formattedStats = () => {
+    const weekTotal = getPeriodTotal('last7Days')
+    const monthTotal = getPeriodTotal('lastMonth')
+
+    if (weekTotal > 4) {
```
Comment (bronson-c) [inline src/common/dataHooks/useStats.js:48]: The stats widget should work the same as when the data was being provided by Nuxeo, however I have been unable tto trigger the widget into displaying "new this week", "new this month" and "on this site" are working correctly though

Comment (sarahfirstvoices) [reply src/common/dataHooks/useStats.js:48]: The API returns "lastWeek" not "last7Days"... does this time period string need to match?

Comment (bronson-c) [reply src/common/dataHooks/useStats.js:48]: That was it! The display is now fixed, thank you!

File: src/components/WidgetStats/WidgetStatsData.js [hunk 2]
```diff
@@ -1,70 +1,9 @@
-import { useQuery } from '@tanstack/react-query'
-
 // FPCC
-import { useSiteStore } from 'context/SiteContext'
-import api from 'services/api'
+import { useStats } from '../../common/dataHooks/useStats'
```
Comment (gmcauliffe) [inline src/components/WidgetStats/WidgetStatsData.js:2]: Avoid using relative imports. You can leverage our Webpack aliases here and change this to `'common/dataHooks/useStats'`

**Review by gmcauliffe — APPROVED** [review]
LGTM! Just one small import change and you're good to merge :)

**Pull request merged**